src/DynamoCoreWpf/ViewModels/Menu/TrustedPathViewModel.cs

```diff
@@ -144,11 +145,13 @@ private void UpdatePathAt(int index)
                 return;
 
             TrustedLocations[index] = args.Path;
+            settings?.SetTrustedLocations(TrustedLocations);
         }
 
         private void RemovePathAt(int index)
         {
             TrustedLocations.RemoveAt(index);
+            settings?.SetTrustedLocations(TrustedLocations);
```

Comment:
@saintentropy LGTM
Maybe call CommitChanges instead. It is the same code and we keep the real change in the same utility.